src/fmu/tools/rms/copy_rms_param_to_ertbox_grid.py

```diff
@@ -1296,29 +1287,29 @@ def check_geogrid_parameters(
     return continuous_type_param_dict, discrete_type_param_list
 
 
-def get_layer_range(
-    indexer: Any, zone_number: int, fmu_mode: bool = False
-) -> tuple[int, int]:
-    _, _, nz = indexer.dimensions
-    if fmu_mode:
-        if len(indexer.zonation) == 1:
-            layer_ranges = indexer.zonation[0]
-        else:
-            raise ValueError(
-                "While in FMU / ERT mode, the grid must have EXACTLY 1 zone"
-            )
-    else:
-        layer_ranges = indexer.zonation[
-            zone_number - 1
-        ]  # Zonation is 0-indexed, while zone numbers are 1-indexed
-    start_layer = nz
-    end_layer = 0
-    for layer_range in layer_ranges:
-        if start_layer > layer_range.start:
-            start_layer = layer_range.start
-        if end_layer < layer_range.stop:
-            end_layer = layer_range.stop
-    return end_layer, start_layer
+# def get_layer_range(
+#    indexer: Any, zone_number: int, fmu_mode: bool = False
+# ) -> tuple[int, int]:
+#    _, _, nz = indexer.dimensions
+#    if fmu_mode:
+#        if len(indexer.zonation) == 1:
+#            layer_ranges = indexer.zonation[0]
+#        else:
+#            raise ValueError(
+#                "While in FMU / ERT mode, the grid must have EXACTLY 1 zone"
+#            )
+#    else:
+#        layer_ranges = indexer.zonation[
+#            zone_number - 1
+#        ]  # Zonation is 0-indexed, while zone numbers are 1-indexed
+#    start_layer = nz
+#    end_layer = 0
+#    for layer_range in layer_ranges:
+#        if start_layer > layer_range.start:
+#           start_layer = layer_range.start
+#        if end_layer < layer_range.stop:
+#            end_layer = layer_range.stop
+#    return end_layer, start_layer
```

Comment:
Instead of commenting out, just remove (git will keep track)

Reply:
yes